contracts/amm/AmmDataProvider.sol

```diff
@@ -608,14 +657,16 @@ contract AmmDataProvider is IAmmDataProvider {
         uint256 wTokenAmount
     ) external view override returns (uint256) {
         IMinterAmm amm = IMinterAmm(ammAddress);
-
         return
             optionTokenGetCollateralOut(
                 seriesId,
                 ammAddress,
                 wTokenAmount,
                 amm.collateralBalance(),
-                getPriceForSeries(seriesId, amm.getVolatility(seriesId)),
+                getPriceForSeries(
+                    seriesController.series(seriesId),
+                    amm.getVolatility(seriesId)
+                ),
```

Comment:
line 667. Is it ok to pass it directly?

Reply:
I believe this is okay @seafi do you have any issues with this here?